Sources/Testing/testMacro.js

```diff
@@ -35,6 +35,7 @@ const DEFAULT_VALUES = {
   myProp11: 11,
   _myProp12: [12],
   _myProp13: 13,
+  myProp14: 14,
```

Comment:
how myProp14 is different from myProp11 ?

Reply:
myProp14 is the same as myProp11 except it does not have any setter. It tests that in this case the `publicAPI.set(...)` call pushes directly to the model the protected property.